src/inflow.c

```diff
@@ -117,38 +108,119 @@ int inflow_readExtInflow(char* tok[], int ntoks)
     {
         basePat = project_findObject(TIMEPATTERN, tok[7]);
         if ( basePat < 0 ) return error_setInpError(ERR_NAME, tok[7]);
-    } 
+    }
+	
+	return(inflow_setExtInflow(j, param, type, tseries, basePat,
```

Comment:
The `swmm_setNodeInflow` API function needs to be validated too. So the data is prepared by either the parser or the API function.. Then validated. .then pushed to the data model.  

The validation step assigns the units conversion factor too (cf).  Why is this important the way it is now? Future expansion.  The `swmm_setNodeInflow` function can be expanded later to allow mass/concentration inflows.  

Honestly, that's about the only thing the validation features does